.github/workflows/ci.yml

```diff
@@ -64,22 +64,9 @@ jobs:
       - name: cargo test (cedar-policy-generators)
         working-directory: ./cedar-policy-generators
         run: cargo test --verbose
-      - name: build.sh
+      - name: Run build.sh script
         shell: bash
         run: source ~/.profile && ./build.sh
```

Comment:
`build.sh` is also run as part of the `run-integration-tests` job/step.  Is this redundant?  Do we need to do it in both places?

Reply:
Technically all that's needed from that script in the `run-integration-tests` jobs is `lake build Cedar:static DiffTest:static Std:static`, but I figured running the whole script in both places would be more consistent. I think it's useful to keep the basic tests separate from the integration tests since the latter may fail for more subtle reasons.

Reply:
Updated the `run-integration-tests` job & task names to avoid the confusion